<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>135</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLineEdit" name="lineEdit_text_veri"/>
     </item>
     <item>
      <widget class="QPushButton" name="buton_veriyi_gonder">
       <property name="text">
        <string>veriyi gonder </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>258</width>
      <height>223</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gelen veriler : </string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>157</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QPushButton" name="buton_yazmaportunu_kapa">
       <property name="text">
        <string>yazma portunu kapat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buton_okuma_portunu_kapat">
       <property name="text">
        <string>okuma portunu kapat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buton_okuma_portunu_ac">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>okuma portunu ac</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_okuma_portu_sec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>okuma portunu seciniz</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buton_yazma_portunu_ac">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>yazma portunu ac</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_yazma_portunu_sec">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>yazma portunu seciniz</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
